js/script.js

```diff
@@ -1,50 +1,79 @@
-let bookLib = JSON.parse(localStorage.getItem('bookLib')) || [];
-function getBook() {
-  const book = {};
-  book.title = document.querySelector('.title').value;
-  book.author = document.querySelector('.author').value;
-  book.id = '';
-
-  if (bookLib === null) {
-    book.id = 0;
-  } else {
-    book.id = bookLib.length;
+class Book {
+  id = 0;
+
+  title = '';
+
+  author = '';
```

Comment:
- [ ] [OPTIONAL] hi team. good job so far, I would suggest to not set the default values for Id, title and author, as this makes it possible to add empty books without any title and author, you can also apply the check when you add the book to the booklist :+1: 
currently :point_down: 
![image](https://user-images.githubusercontent.com/97521557/228314429-25c7aa1b-8546-482d-a9d1-1ca2e5d74b40.png)